### PR TITLE
Create functionality

### PR DESCRIPTION
**Pull request by jayfr1122**

### CODE REVIEW

File: app/controllers/apartments_controller.rb [hunk 1]
```diff
@@ -1,2 +1,24 @@
 class ApartmentsController < ApplicationController
+  def index
+  apartments = Apartment.all
+  render json: apartments
+end
+  apartments = Apartment.all
+  render json: apartments
```
Comment (elysemontano) [inline app/controllers/apartments_controller.rb]: This looks like it doesn't belong here.

File: app/controllers/apartments_controller.rb [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+    render json: apartment.errors, status: 422
+  end
+end
+
+private
+def apartment_params
+  params.require(:apartment).permit(:street, :city, :state, :unit, :square_footage, :price, :bedrooms, :bathrooms, :pets, :image, :user_id)
+end
+
+end
```
Comment (elysemontano) [inline app/controllers/apartments_controller.rb]: Double check indentation on this entire file.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Tests look great. Left a few comments in your controller to adjust then you are good to go!

File: app/controllers/apartments_controller.rb [hunk 3]
```diff
@@ -1,2 +1,18 @@
...
+    if apartment.valid?
+     render json: apartment
+    else
+     render json: apartment.errors, status: 422
+  end
 end
+private
+  def apartment_params
+    params.require(:apartment).permit(:street, :city, :state, :unit, :square_footage, :price, :bedrooms, :bathrooms, :pets, :image, :user_id)
+  end
```
Comment (elysemontano) [inline app/controllers/apartments_controller.rb]: This should still be within the scope of the class.  The end on line 13 should be after your strong params

**Review by elysemontano — CHANGES_REQUESTED** [review]
Left a comment in controller

File: app/controllers/apartments_controller.rb (hunk 3 above)
Comment (jayfr1122) [reply app/controllers/apartments_controller.rb]: making appropriate changes

**Review by elysemontano — APPROVED** [review]

**Pull request merged**